docs/en/integrations/data-ingestion/data-formats/csv-tsv.md

```diff
@@ -0,0 +1,377 @@
+---
+sidebar_label: CSV and TSV
+sidebar_position: 1
+slug: /en/integrations/data-formats/csv-tsv
+---
+
+# Working with CSV and TSV data in ClickHouse
+
+ClickHouse supports importing data from and exporting to CSV. Since CSV files can come with different format specifics, including header rows, custom delimiters, and escape symbols, ClickHouse provides formats and settings to address each case efficiently.
+
+
+## Importing data from a CSV file
+
+Before importing data, let’s create a table with a relevant structure:
+
+```sql
+CREATE TABLE sometable
+(
+    `path` String,
+    `month` Date,
+    `hits` UInt32
+)
+ENGINE = MergeTree
+ORDER BY tuple(month, path)
+```
+
```

Comment:
could we maybe populate this e.g. from an s3 or HTTP link so users can actually run examples?

Reply:
Done